<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCRFlashcard</class>
 <widget class="QDockWidget" name="OCRFlashcard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="allowedAreas">
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <property name="windowTitle">
   <string>OCRFlashcard</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QListWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>256</width>
      <height>391</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addFileButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>475</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileQueueTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8080FF</string>
    </property>
    <property name="text">
     <string>File Queue</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="progressTitle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>281</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8080FF</string>
    </property>
    <property name="text">
     <string>Progress</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="flashcardOptionsTitle">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>216</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8080FF</string>
    </property>
    <property name="text">
     <string>Flashcard Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="curWorkingFileLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>281</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Current Working File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="curWorkingFileDisplay">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>281</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="numFileRemDisplay">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>281</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fileRemainingLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>281</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Files Remaining</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="flashCreatedLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>281</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Flashcards Created</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="kanjiFlashcardOptionsLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>51</y>
      <width>189</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Kanji Flashcard Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="wordFlashcardOptionsLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>158</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Word Flashcard Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sentenceFlashcardOptionsLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>303</y>
      <width>222</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sentence Flashcard Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ocrOptionsTitle">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>320</y>
      <width>256</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8080FF</string>
    </property>
    <property name="text">
     <string>OCR Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="setCropBoundaryButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>475</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Crop Boundary</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="selectedFilesRadiobutton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>For selected file(s)</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">ocrOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="allFilesRadiobutton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>131</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>For all files</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">ocrOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="flashCreatedSentenceLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>141</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Sentence</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="flashCreatedWordLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>141</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Word</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="flashCreatedKanjiLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>141</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Kanji</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="flashCreatedSentenceDisplayLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>81</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="flashCreatedWordDisplayLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>280</y>
      <width>81</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="flashCreatedKanjiDisplayLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>81</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="kanjiRecallRecogRadiobutton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>78</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Recall + Recognition</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">kanjiFlashcardOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="kanjiRecogRadiobutton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>105</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Recognition</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">kanjiFlashcardOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="wordRecogRadiobutton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>213</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Recognition</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">wordFlashcardOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="wordRecallRecogRadiobutton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>186</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Recall + Recognition</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">wordFlashcardOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="sentenceFlashcardClozeLabel">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>331</y>
      <width>198</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Create a cloze deletion card for...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="sentenceRandomWordsRadiobutton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>353</y>
      <width>175</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>             Random words</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sentenceFlashcardOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="sentenceEachWordRadiobutton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>379</y>
      <width>92</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Each word</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sentenceFlashcardOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="sentenceNoneRadiobutton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>406</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">sentenceFlashcardOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="kanjiNoneButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>132</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">kanjiFlashcardOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="wordNoneRadiobutton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>239</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">wordFlashcardOptionsGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="removeFileButton">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>475</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Files</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>475</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusTitle">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>430</y>
      <width>81</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #8080FF</string>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>475</y>
      <width>84</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="curStatusLabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>437</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Idle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="randomNumInput">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>350</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="inputMask">
     <string notr="true">999</string>
    </property>
    <property name="text">
     <string>003</string>
    </property>
   </widget>
   <widget class="QPushButton" name="swapTextDirectionButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>425</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Swap Text Direction</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileQueueFormatLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>256</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>File Name / Text Direction / Boundary</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ocrOptionsWarningLabel">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>350</y>
      <width>331</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Only image and PDF files will have boundary applied.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="templateSelection">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>385</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ocrOptionsTemplateLabel">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>382</y>
      <width>111</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Template Image:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="allowDupesCheckbox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>261</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow duplicate words in 
separate sentences</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="enhanceCheckbox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>485</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enlarge Images</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sentenceFlashcardOptionsGroup"/>
  <buttongroup name="wordFlashcardOptionsGroup"/>
  <buttongroup name="ocrOptionsGroup"/>
  <buttongroup name="kanjiFlashcardOptionsGroup"/>
 </buttongroups>
</ui>
